microservice-vote/pom.xml

```diff
@@ -331,6 +338,22 @@
                             <goal>install-server</goal>
                         </goals>
                     </execution>
+                    <!-- Install the config feature into the liberty server -->
+               		<execution>
+           				<id>install-feature</id>
+           				<phase>prepare-package</phase>
+          				<goals>
+               				<goal>install-feature</goal>
+           				</goals>
+          				<configuration>
+             		  	  	<features>
+               		    		<acceptLicense>true</acceptLicense>
+                  	 			<feature>mpConfig-1.0</feature>
```

Comment:
update to
 `
<feature>mpConfig-1.1</feature>
`

Reply:
@Emily-Jiang Liberty 17.9.0.0 only provides mpConfig-1.0, so we cannot update to mpConfig-1.1 yet.